rtf_vis_tool/.eslintcache

```diff
@@ -0,0 +1 @@
mn":16},{"ruleId":"114","severity":1,"message":"119","line":13,"column":8,"nodeType":"116","messageId":"117","endLine":13,"endColumn":26},{"ruleId":"114","severity":1,"message":"120","line":34,"column":12,"nodeType":"116","messageId":"117","endLine":34,"endColumn":19},{"ruleId":"114","severity":1,"message":"121","line":69,"column":11,"nodeType":"116","messageId":"117","endLine":69,"endColumn":35},{"ruleId":"122","severity":1,"message":"123","line":79,"column":27,"nodeType":"124","endLine":79,"endColumn":105},{"ruleId":"114","severity":1,"message":"125","line":1,"column":16,"nodeType":"116","messageId":"117","endLine":1,"endColumn":25},{"ruleId":"114","severity":1,"message":"126","line":1,"column":37,"nodeType":"116","messageId":"117","endLine":1,"endColumn":43},{"ruleId":"127","severity":1,"message":"128","line":70,"column":75,"nodeType":"129","messageId":"130","endLine":70,"endColumn":77},{"ruleId":"131","severity":1,"message":"132","line":14,"column":65,"nodeType":"133","messageId":"134","endLine":14,"endColumn":97},{"ruleId":"131","severity":1,"message":"132","line":15,"column":43,"nodeType":"133","messageId":"134","endLine":15,"endColumn":78},{"ruleId":"114","severity":1,"message":"135","line":2,"column":5,"nodeType":"116","messageId":"117","endLine":2,"endColumn":8},{"ruleId":"114","severity":1,"message":"126","line":1,"column":37,"nodeType":"116","messageId":"117","endLine":1,"endColumn":43},{"ruleId":"114","severity":1,"message":"136","line":6,"column":8,"nodeType":"116","messageId":"117","endLine":6,"endColumn":18},{"ruleId":"137","severity":1,"message":"138","line":37,"column":8,"nodeType":"139","endLine":37,"endColumn":10,"suggestions":"140"},{"ruleId":"141","severity":1,"message":"142","line":17,"column":32,"nodeType":"143","messageId":"144","endLine":17,"endColumn":34},"no-unused-vars","'Canvas' is defined but never used.","Identifier","unusedVar","'PCViewer' is defined but never used.","'TranSuccessSummary' is defined but never used.","'showTSS' is assigned a value but never used.","'toggleTranSummaryDisplay' is assigned a value but never used.","react/jsx-pascal-case","Imported JSX component Data_Association_PC must be in PascalCase or SCREAMING_SNAKE_CASE","JSXOpeningElement","'useEffect' is defined but never used.","'useRef' is defined but never used.","eqeqeq","Expected '===' and instead saw '=='.","BinaryExpression","unexpected","no-throw-literal","Expected an error object to be thrown.","ThrowStatement","object","'SE3' is assigned a value but never used.","'SpriteMesh' is defined but never used.","react-hooks/exhaustive-deps","React Hook useEffect has a missing dependency: 'props.json'. Either include it or remove the dependency array.","ArrayExpression",["145"],"array-callback-return","Array.prototype.map() expects a return value from arrow function.","ArrowFunctionExpression","expectedInside",{"desc":"146","fix":"147"},"Update the dependencies array to be: [props.json]",{"range":"148","text":"149"},[1598,1600],"[props.json]"]
```

Comment:
these look like files specific to your system Adi -- do we need them?
```python
"C:\\Users\\adisi\\Desktop\\Georgia Tech\\Spring 2021\\SfM Research\\gtsfm\\rtf_vis_tool\\src\\Components\\SfMDataDivNode.js"
```

Reply:
Nope. I'll add this to my .gitignore